lib/chef/knife/ec2_server_create.rb

```diff
@@ -418,6 +418,12 @@ class Ec2ServerCreate < Knife
         :description => "Comma-separated list of security group ids for ClassicLink",
         :proc => Proc.new { |groups| groups.split(',') }
 
+      option :disable_api_termination,
+        :long => "--disable-api-termination",
+        :description => "Cannot terminate the instance using the Amazon EC2 console, CLI and API.",
```

Comment:
"Disable termination of the instance using the Amazon EC2 console, CLI, and API."